src/Sylius/Bundle/RbacBundle/spec/Form/Type/PermissionTypeSpec.php

```diff
@@ -46,17 +47,18 @@ function it_should_build_form_with_proper_fields(FormBuilder $builder)
         ;
 
         $builder
-            ->add('parent', 'sylius_permission_choice', Argument::any())
+            ->addEventSubscriber(Argument::type(AddCodeFormSubscriber::class))
             ->shouldBeCalled()
```

Comment:
I'm not conviced to create spec which checks nothing. We need to be sure that our builder really add subscribers and form field which are expected.